components/ListView/ListItemUploads.js

```diff
@@ -46,6 +46,15 @@ const messages = defineMessages({
 class ListItemUploads extends React.PureComponent {
   constructor(props) {
     super(props);
+    this.uploadProviderToLabel = this.uploadProviderToLabel.bind(this);
+  }
+
+  uploadProviderToLabel(uploadType) {
+    const uploadTypeLabels = {
+      aws: "AWS",
+      azure: "Azure",
+    };
+    return uploadTypeLabels[uploadType];
```

Comment:
This will throw an error if a new upload type is ever introduced in the API that isn't listed here. Could we future-proof this a bit, and just return the raw `uploadType` value if a label isn't defined for that upload type? We have something similar currently for the compose types.

Reply:
This is fixed. If uploadTypeLabels[uploadType] is undefined I will return the provided uploadType.